.pre-commit-config.yaml

```diff
@@ -18,9 +18,11 @@ repos:
     hooks:
     -   id: remove-tabs
 -   repo: https://github.com/sceptre/sceptrelint
-    rev: v1.0.1
+    rev: v1.1.0
     hooks:
     -    id: check-file-names
     -    id: check-stack-names
-    -    id: check-stack-tags
-         args: [--tag=CostCenter]
+    -    id: check-stack-tag-values
+         args: [--tag=CostCenter,
+                --file=https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/tags/SageFinancialProgramCodes.json,
```

Comment:
Validation should either (1) disallow CostCenter=Other / 000001 or (2) allow  CostCenter=Other / 000001 but then also require that there is a CostCenterOther tag with a value other than Other / 000001.

Reply:
going with option 1.